bundler/lib/bundler/cli/common.rb

```diff
@@ -97,5 +97,21 @@ def self.clean_after_install?
       clean &&= !Bundler.use_system_gems?
       clean
     end
+
+    protected
+
+    def self.word_list(words)
+      if words.empty?
+        return ''
+      end
+
+      words = words.map { |word| "'#{word}'" }
+
+      if words.length == 1
+        return words[0]
+      end
+
+      [words[0..-2].join(", "), words[-1]].join(" or ")
```

Comment:
```suggestion
      [words[0..-2].join(", "), words[-1]].join(", or ")
```

Personally I like oxford commas ("apple, orange, or pear"). I went without it so the output is similar to how it has always been ("apple, orange or pear").